iroha/iroha.py

```diff
@@ -341,19 +341,26 @@ class IrohaGrpc(object):
     Possible implementation of gRPC transport to Iroha
     """
 
-    def __init__(self, address=None, timeout=None, secure=False):
+    def __init__(self, address=None, timeout=None, secure=False, max_message_length=5 * 1024**3):
         """
         Create Iroha gRPC client
         :param address: Iroha Torii address with port, example "127.0.0.1:50051"
         :param timeout: timeout for network I/O operations in seconds
         :param secure: enable grpc ssl channel
+        :param max_message_length: it is max message length in bytes for grpc, default 5Mb
         """
         self._address = address if address else '127.0.0.1:50051'
 
```

Comment:
```python
channel_kwargs = {}
if max_message_length is not None:
  channel_kwargs['options'] = [(...), ...]
```

Reply:
this assumes the default `max_message_length` is `None`

Reply:
I've changed as You suggested. I didn't know about the possibility in python.